src/System.ComponentModel.TypeConverter/tests/LicenseManagerInteropTests.cs

```diff
@@ -1,42 +0,0 @@
-// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
Are there other tests that exist for this?  Do we need to add more?

Reply:
@stephentoub These are going to be in coreclr now since that will be able to test the complete E2E. If we really want tests in CoreFX as well, I can bring this back, but the coreclr tests will exercise all APIs in this review.